tests/api/api.spec.ts

```diff
@@ -395,6 +400,23 @@ suite("@dataform/api", () => {
       expect(actionNames).includes("schema.tab_a");
     });
 
+    test("prune actions with --tags (with include all tags)", () => {
+      const prunedGraph = prune(TEST_GRAPH_WITH_TAGS, {
+        tags: ["tag1", "tag2"],
+        includeAllTags: true
+      });
+      const actionNames = [
+        ...prunedGraph.tables.map(action => targetAsReadableString(action.target)),
+        ...prunedGraph.operations.map(action => targetAsReadableString(action.target))
+      ];
+      expect(actionNames).includes("schema.op_e");
+      expect(actionNames).includes("schema.tab_a");
+      expect(actionNames).not.includes("schema.op_a");
+      expect(actionNames).not.includes("schema.op_b");
+      expect(actionNames).not.includes("schema.op_c");
+      expect(actionNames).not.includes("schema.op_d");
```

Comment:
This would be better as `expect(actionNames).deep.equals(["schema.op_e", "schema.op_a"]);`